src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/ModalBox.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import React from 'react';
+import Icon from '../Icon';
+import type {ModalProps} from './types';
+import Actions from './Actions';
+import modalBoxStyles from './modalBox.scss';
+
+type Props = ModalProps & {
+    onRequestClose: () => void,
+}
+
+const CLOSE_ICON = 'times';
+
+export default class ModalBox extends React.PureComponent<Props> {
+    static defaultProps = {
+        actions: [],
+    };
+
+    render() {
+        return (
+            <div className={modalBoxStyles.box}>
+                <header>
+                    {this.props.title}
+                    <Icon
+                        name={CLOSE_ICON}
+                        className={modalBoxStyles.icon}
+                        onClick={this.props.onRequestClose} />
+                </header>
+                <section>
```

Comment:
I would have left this one a `div` and the outer `div` would be the `section` for me, since a `section` is allowed to have `headers` and `footers` (if I am not mistaken).